lib/ancestry/instance_methods.rb

```diff
@@ -100,7 +100,7 @@ def ancestry_changed?
     end
 
     def parse_ancestry_column obj
-      obj.to_s.split('/').map { |id| cast_primary_key(id) }
+      obj.to_s.split(self.ancestry_base_class.ancestry_delimiter).map { |id| cast_primary_key(id) }
```

Comment:
@stefankroes I tried to follow the convention in place for whether calling `ancestry_base_class.X` vs `get_X`, but I don't understand the reason for the 3 different ways to access the same values.

Also, the class level variables goes against allowing multiple trees per node, which you had expressed interest. (note: I did not say multiple parents per node.)

I'm thinking about going through and consolidating the number of different ways these are accessed. any comments or advice?